### ISSUE TITLE
Create a reference for Ewing samples to use with `SingleR`

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/OpenScPCA-analysis#607]
### If you are filing this issue based on a specific GitHub Discussion, please link to the relevant Discussion.

#292 

### Describe the goals of the changes to the analysis module.

We want to be able to run `SingleR` on the remaining Ewing samples that have been difficult to annotate based on marker gene expression alone. Before we can do this we need to create a trained model from our reference samples. We will use the tumor cell assignments from SCPCL000822, SCPCL000824, and SCPCL001114 along with a normal cell reference from `celldex`. 

### What will your pull request contain?

The first thing we will need to do is add in the tumor cell annotations to the SCE object. 
Then we will finalize the annotations stored in the individual reference SCE objects. 

- Tumor cells will be labeled as "Tumor"
- Any non tumor cells that are labeled with the same cell type in using both SingleR and CellAssign will get that assignment (using the original scpca-nf annotations). 
- All other cells will be re-annotated using `BlueprintEncodeData` from `celldex`

Then the `SingleR` model will be created using the three samples and Blueprint and saved to an RDS object to use later to annotate all other samples. 

### Will you require additional software beyond what is already in the analysis module?

No

### Will you require different computational resources beyond what the analysis module already uses?

No

**Comment by allyhawkins** [comment]
After talking about this some more, we are going to take a slightly different approach. 
We feel confident that the tumor cells being called by `AUCell` and through our marker gene methods are in fact tumor cells, but we are worried that we are calling many cells normal that are in fact tumor cells. So we are going to take the following approach: 

1. Obtain confident tumor cell annotations for every library. These are going to come from the output of `aucell-annotation.sh` so that we have annotated tumor cells consistently across all samples. I plan to update that output to also include a consensus column that takes the consensus between AUCell and the marker gene cut off of expression at least 2 marker genes. (I will file a separate issue to address these updates)
2. Next we will create a reference that contains the confident tumor cells from all samples in SCPCP000015. This will be a merged object that contains the gene expression for all tumor cells from all samples and a column in the `colData` that annotates the cells with `tumor-library_id` so that we can identify which tumor cells came from which library. 
3. Then we will write a script that runs `SingleR` on each library in the project. For each project, we will remove the tumor cells from that library in the reference and then combine the remaining tumor cells with `BlueprintEncodeData` from `celldex`. This will then return annotations of the remaining non-confident cells. This will tell us if the remaining cells are more similar to the other tumor cell profiles or to profiles of normal cells. 

I'll keep this issue open to track creating the reference (step 2), but will file separate issues for 1 and 3.

**Comment by allyhawkins** [comment]
> Obtain confident tumor cell annotations for every library. These are going to come from the output of aucell-annotation.sh so that we have annotated tumor cells consistently across all samples. I plan to update that output to also include a consensus column that takes the consensus between AUCell and the marker gene cut off of expression at least 2 marker genes. (I will file a separate issue to address these updates)

In thinking about this more, I actually don't think we need the consensus. In our initial exploration in #612, we see that all cells called tumor by `AUCell` are also called tumor cells by marker genes. Taking the cells called tumor by `AUCell` has been showing to be the most conservative method. It relies on having at least some marker gene expression to make the call as tumor so I think that's sufficient. Because of that I think we can just use the current output of `aucell-annotation.sh`. to address # 1.
